k8s/default-namespace/README.md

```diff
@@ -0,0 +1,15 @@
+## Kubernetes Files for Versions 2.1.1 and Up
+
+This directory contains the manifest files used for installing the Scalyr Agent prior to the 2.1 Scalyr Agent release. You should only use these manifest files if you are running an instance prior to 2.1.
```

Comment:
Insert this text after the first sentence and delete the second sentence:

Prior to 2.1, the Scalyr Agent K8s instructions created all objects in the default namespace.  This has since changed to create all objects in the scalyr namespace.  You should only use these manifest files if you need to continue running the Scalyr Agent in the default namespace, possibly as a stopgap measure until you can migrate your configuration.